portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,12 +25,30 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+   
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String quote="Hello Pranjali!";
-   
-    response.setContentType("text/html;");
+    //Create ArrayList<String> object
+    ArrayList<String> list= new ArrayList<String>();   
+    list.add("And I knew exactly what to do. But in a much more real sense, I had no idea what to do.- Michael G. Scott");
+    list.add("It is a capital mistake to theorize before one has data. Insensibly one begins to twist facts to suit theories, instead of theories to suit facts.- Sherlock Holmes");
+    list.add("When you are backed against the wall, break the goddamn thing down.- Harvey Specter");
+    list.add("Never forget what you are. The rest of the world will not. Wear it like armor, and it can never be used to hurt you.- Tyrion Lannister");
+    //Convert object to JSON
+    String quote=convertToJsonUsingGson(list);
```

Comment:
Add space before and after the '='. Run the linter I shared with you and that should take care of formatting issues like this.

Maybe name the variable quotesAsGson or something like that or remove the variable completely and just inline it below like this:
response.getWriter().println(convertToJsonUsingGson(...))